src/sockets/reactnative.ts

```diff
@@ -0,0 +1,64 @@
+import { ConnectionOptions, Data, Socket } from '.';
```

Comment:
Nit but could the filename be `react-native` -- that's what we use for the other file paths like `drivers/react-native-sqlite-storage` and the `electric-sql/react-native` export.